Library/Homebrew/cmd/test-bot.rb

```diff
@@ -91,6 +92,10 @@ def failed?
 
     def puts_command
       cmd = @command.join(" ")
+      if ENV["TRAVIS"]
+        puts cmd
```

Comment:
Looks like we don't need `puts cmd` here. https://travis-ci.org/Homebrew/homebrew/jobs/80807853